docs/compiler-guide.md

```diff
@@ -577,10 +580,12 @@ These internal parts tend to be implemented with the "module containing lots of
 
 ### The F# Compiler Service Operations Queue
 
+TODO: where should this link point?
```

Comment:
This should go here: https://fsharp.github.io/FSharp.Compiler.Service/queue.html